Lesson Plans/Flight Instructor/CFI/CFI1.4 Teaching Process.md

```diff
@@ -0,0 +1,250 @@
+---
+tags:
+  - cfi
+  - lesson
+---
+Related to [[Steps in the Teaching Process]] and [[Elements of the Teaching Process]]
+
+### Objective
+The learner acquires knowledge and skills related to course & lesson development for aviation instruction.
+
+### Resources
+- [[CFI ACS]] Section FI.I.C
+- [[AIH Ch5]]
+- [[AIH Ch7]]
+
+### Builds On
+None
+
+### Schedule
+Ground 0.5 hours
+
+### Lesson Elements
+
+#### Teaching
+1. Process
+	1. Organizes the material in a way that the learner can understand
+	2. Preparation, Presentation, Application and Assesment
+2. Essential Teaching Skills
+	1. People Skills
+		1. Technical knowledge is useless if not communicated effectively
+		2. Effective communication is two-way and includes active listening to the learner
+		3. Challenge the learner while supporting their efforts
+	2. Subject Matter Expertise
+		1. Effective instructor have a sincere interesting in learning & professional growth
+		2. Networking and observing other instructors and seeking mentoring is also helpful; Be a lifelong learner
+	3. Management Skills
+		1. Ability to plan, organize, lead and supervise (a lesson)
+		2. Manage the time effectively, plan for the unexpected
+		3. Aviation instructor also need to supervise learners (ex. preflight procedures)
+	4. Assessment Skills
+		1. Measuring the progress (behavior change)
+		2. Purpose of the assessment should be made clear
+
+#### [[Flight Instructor Model Code of Conduct]]
```

Comment:
I'm confused by this one -- is it meant to be part of the "#### Teaching" section? Or is it it's own thing? 

I guess what I mean is when I read the lesson outline, I'm not sure how it flows together.

Reply:
lol fair, I was following the original flow in AIH but agree it does not make much sense to be in the middle: I think it's just they want/need to talk about code of conducts as an instructor before diving into teaching. 

<img width="438" height="812" alt="image" src="https://github.com/user-attachments/assets/b7403656-0292-45ba-be89-512e6813d3ac" />

How about this: I'll promote this to the top of the lesson. Open the lesson with responsibilities of being an instructor.